readPass/index.js

```diff
@@ -67,50 +67,54 @@ exports.handler = async (event, context) => {
       const passData = await runQuery(queryObj);
       console.log("passData", passData);
 
-      // Build cancellation email payload
-      const claims = {
-        iss: 'bcparks-lambda',
-        sub: 'readPass',
-        passId: event.queryStringParameters.passId,
-        parkName: event.queryStringParameters.park
-      }
-      const token = jwt.sign(claims, process.env.JWT_SECRET, { expiresIn: '15m' });
-
-      const cancellationLink = process.env.PUBLIC_FRONTEND
-      + process.env.PASS_CANCELLATION_ROUTE
-      + "?passId=" + passData.data[0].registrationNumber
-      + "&email=" + passData.data[0].email
-      + "&code=" + token;
-
-      const encodedCancellationLink = encodeURI(cancellationLink);
-
-      let personalisation =  {
-        'registrationNumber': passData.data[0].registrationNumber.toString(),
-        'link': encodedCancellationLink
-      };
-
-      // Send email
-      // Public page after 200OK should show 'check your email'
-      try {
-        const emailRes = await axios({
-          method: 'post',
-          url: process.env.GC_NOTIFY_API_PATH,
-          headers: {
-            'Authorization': process.env.GC_NOTIFY_API_KEY,
-            'Content-Type': 'application/json'
-          },
-          data: {
-            'email_address': passData.data[0].email,
-            'template_id': process.env.GC_NOTIFY_CANCEL_TEMPLATE_ID,
-            'personalisation': personalisation
-          }
-        });
-
-        return sendResponse(200, personalisation);
-      } catch  (err) {
-        let errRes = personalisation;
-        errRes["err"] = "Email Failed to Send";
-        return sendResponse(200, errRes);
+      if (passData && passData.data && passData.data.length !== 0) {
+        // Build cancellation email payload
+        const claims = {
+          iss: 'bcparks-lambda',
+          sub: 'readPass',
+          passId: event.queryStringParameters.passId,
+          parkName: event.queryStringParameters.park
+        }
+        const token = jwt.sign(claims, process.env.JWT_SECRET, { expiresIn: '15m' });
+
+        const cancellationLink = process.env.PUBLIC_FRONTEND
+          + process.env.PASS_CANCELLATION_ROUTE
+          + "?passId=" + passData.data[0].registrationNumber
+          + "&email=" + passData.data[0].email
+          + "&code=" + token;
+
+        const encodedCancellationLink = encodeURI(cancellationLink);
+
+        let personalisation =  {
+          'registrationNumber': passData.data[0].registrationNumber.toString(),
+          'link': encodedCancellationLink
+        };
+
+        // Send email
+        // Public page after 200OK should show 'check your email'
+        try {
+          const emailRes = await axios({
```

Comment:
No need to define anything here since we're not using it.  Just await axios is fine.